packages/zipkin/src/instrumentation/httpServer.js

```diff
@@ -85,6 +85,8 @@ class HttpServerInstrumentation {
     this.tracer.recordBinary('http.status_code', statusCode.toString());
     if (error) {
       this.tracer.recordBinary('error', error.toString());
+    } else if (statusCode < 200 || statusCode > 399) {
```

Comment:
This is the piece of code doing the trick for http servers.

Reply:
Looks good to me